content/_includes/page/meta.njk

```diff
@@ -1,16 +1,19 @@
 {%- import 'utility.macros.njk' as utility -%}
+{%- import 'tags.macros.njk' as tagutils -%}
 
 {% set public_tags = tags | publicTags %}
 {% set elsewhere = {
   'venue': venue,
   'url': url
 } if elsewhere else none %}
 
+{% set sub = renderData.sub or sub %}
+
 {% if sub or public_tags or elsewhere or author %}
   <div id="meta">
     {% if sub or dates %}
       <p class="subtitle">
-        {{ sub | mdInline('amp') | safe }}
+        {{ sub | mdInline | safe }}
```

Comment:
`amp` is one of several typography helpers we have, to ensure we use our fancy ampersand. I had a special-case for applying it without the other typography settings, but decided to update `mdInline` so it would handle this automatically.